src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -1843,6 +1844,43 @@ def test_fw_call_node(fortran_writer):
     assert expected in result
 
 
+def test_fw_intrinsic_output_control(fortran_writer):
+    '''Test the config controls the output of IntrinsicCall nodes correctly.
+    '''
+    args = [Reference(DataSymbol("arg1", REAL_TYPE)),
+            Reference(DataSymbol("arg2", REAL_TYPE))]
+    call = IntrinsicCall.create(IntrinsicCall.Intrinsic.ISHFT,
+                                args)
+    result = fortran_writer(call)
+    assert "ISHFT(i=arg1, shift=arg2)" in result
+
+    # Turn off the output of required arguments
+    Config.get().intrinsic_kwargs = False
```

Comment:
I think this needs a slightly more descriptive name. Perhaps `backend_intrinsic_named_kwargs`? It's a bit wordy. Perhaps one day we need to support some sort of hierarchy in the Config object (e.g. a section for the frontend, for the backend, for ...).

Reply:
Renamed.